impargo/react-here-maps-old-fork
# refactor: switch to functional components and support react 18

*Testing procedures*
1. Somehow get it to run in the CA
2. Test map functionality in the planner (stop markers, context menu, favorites, layers)
3. Test map functionality in fleet (markers, popups)
4.
Note: there will be a couple of TypeScript errors because of the change in types from class components to functional components. But functionality should not be broken
